cli/src/meta/mod.rs

```diff
@@ -35,7 +37,9 @@ impl TryFrom<KnownMagic> for KnownMeta {
 pub enum ContentType {
     None,
     #[serde(rename = "application/json")]
-    Json
+    Json,
+    #[serde(rename = "application/cbor")]
```

Comment:
can we test this?

Reply:
already tested

Reply:
added a test in cli/src/cli/build.rs file